config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-config.yaml

```diff
@@ -605,6 +610,11 @@ presubmits:
         base_ref: release-1.7
         path_alias: sigs.k8s.io/cluster-api-provider-azure
         workdir: true
+      - org: kubernetes-sigs
+        repo: cloud-provider-azure
+        base_ref: release-1.25
```

Comment:
should this be release-1.26?

Reply:
KUBERNETES_VERSION is set to "latest-1.25" (i.e. release-1.25 branch) which means we should test with release-1.25 CCM according to this version matrix: https://github.com/kubernetes-sigs/cloud-provider-azure#version-matrix

https://github.com/kubernetes/test-infra/pull/28952/files#diff-35b54403079e54ae55255f095a84af44fa2d1c2de8e4b10a010861fa09dd7fc4L632

Reply:
Ah, I was doing the wrong correlation (I was correlating the azuredisk version --> ccm version)

Reply:
yeah I was also confused about the k8s versions we're using, hence my question to @andyzhangx above https://github.com/kubernetes/test-infra/pull/28952/files#r1128609688

but in any case CCM version needs to match k8s version